src/lib/overlay-element.ts

```diff
@@ -0,0 +1,98 @@
+import { EditableStore } from './editable-store.ts';
+import { OverlayManager } from './overlay-manager.ts';
+
+export class OverlayElement {
+	private noDimensions: boolean = false;
```

Comment:
Renaming this to `hasNoDimensions` might more sense than `noDimensions` and will more closely match the structure of `hasDimensions`